controllers/fenceagentsremediation_controller_test.go

```diff
@@ -289,10 +391,28 @@ func cleanupTestedResources(va1, va2 *storagev1.VolumeAttachment, pod *corev1.Po
 	podTest := &corev1.Pod{}
 	if err := k8sClient.Get(context.Background(), client.ObjectKeyFromObject(pod), podTest); err == nil {
 		log.Info("Cleanup: clean pod", "pod name", podTest.Name)
-		Expect(k8sClient.Delete(context.Background(), podTest)).To(Succeed())
+		var grace client.GracePeriodSeconds = 0
```

Comment:
Can it be a const then it will be used in many functions and not per function whenever we delete a pod?